go.mod

```diff
@@ -34,4 +33,6 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6 // avoids case-insensitive import collision: "github.com/googleapis/gnostic/openapiv2" and "github.com/googleapis/gnostic/OpenAPIv2"
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/alexeykazakov/toolchain-common v0.0.0-20201208030835-d24a52f72536
```

Comment:
Reminder to myself to remove it when https://github.com/codeready-toolchain/toolchain-common/pull/143 is merged.

Reply:
done 47a7386